waltz-jobs/src/main/java/com/khartec/waltz/jobs/clients/c1/sc1/model/ProjectRow.java

```diff
@@ -0,0 +1,59 @@
+package com.khartec.waltz.jobs.clients.c1.sc1.model;
+
+import com.khartec.waltz.jobs.Columns;
```

Comment:
Ditto, this file is just for sample data generation.

Reply:
Noticed this but figured it as an example